Homework3/task_1.py

```diff
@@ -0,0 +1,30 @@
+# Задача 16: 
+# Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+# *Пример:*
+# 5
+#     1 2 3 4 5
+#     3
+#     -> 1
+
+from random import randint
+
+while True:
+    n = int(input('Задайте количество элементов массива: '))
+    if n <= 0:
+        print('Количество элементов должно быть натуральным числом.')
+    else:
+        break
+
+lst = [randint(-10, 10) for _ in range(n)]
+print(f"Массив: {lst}")
+
+k = int(input('Задайте целое число в диапазоне от -10 до 10: '))
+
+count = 0
+for i in range(n):
+    if lst[i] == k:
```

Comment:
Функцию `range` использовать совершенно не обязательно, можно напрямую проходиться по элементам списка.
```suggestion
for i in lst:
    if i == k:
```